docs/api-testing/integrations/pagerduty-webhooks.md

```diff
@@ -75,11 +69,11 @@ To test a connector:
 2. Click on any project.
 3. Within that project, click **Settings** and then click **Connector**.
 4. On the **Connectors** page, click **Test Connectors**.
-<img src={useBaseUrl('img/api-testing/test-connectors-nav.png')} alt="Test Connectors button" width="600"/>
```

Comment:
@LoryApiFortress Did you remove the image sizing here? Without the parameter the image is too big. I adjusted this in a PR I merged this morning.